pkg/cmd/provisioning/utils.go

```diff
@@ -1,10 +1,51 @@
 package provisioning
 
 import (
+	"crypto"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
 	"fmt"
+	"gopkg.in/square/go-jose.v2"
+	"io/ioutil"
+	"log"
 	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+var (
```

Comment:
store this in utils.go file is not great. store these in a file that provides info that it has functions for oidc discovery related functions

Reply:
How about moving it to the common constants file? This template is technically a constant, common to all the cloud providers.

Reply:
moved discovery document template to common constants file